test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -143,7 +144,13 @@ protected virtual void CreateTestStorageEntities()
 
         public virtual void Dispose()
         {
-            Host.Stop();
+            try
+            {
+                Host.Stop();
+            }
+            catch
+            {
+            }
```

Comment:
We need to do some careful e2e testing verifying all this - the risk is that incomplete host shutdown would cause us to leak listeners and their expensive resources.